spec/README.md

```diff
@@ -0,0 +1,22 @@
+Jakarta WebSocket Specification
+============================
+
+This project generates the Jakarta WebSocket Specification.
+
+Building
+--------
+
+Prerequisites:
+
+* JDK8+
```

Comment:
If this is to remain Java EE 8 compatible, then this JDK8 requirement is odd.
What is this JDK8 requirement for?  the API or the Spec?
(API has traditionally had a JDK6+ requirement for releases 1.0 and 1.1)